contracts/test/nft_market_case_1.test.ts

```diff
@@ -0,0 +1,138 @@
+import { $log } from '@tsed/logger';
+import { BigNumber } from 'bignumber.js';
+import { TezosToolkit, MichelsonMap } from '@taquito/taquito';
+import { InMemorySigner } from '@taquito/signer';
+
+import { bootstrap, TestTz } from './bootstrap-sandbox';
+import { Contract, nat, address, bytes } from '../src/type-aliases';
+
+import {
+    originateNftFaucet,
+    originateNft,
+    MintNftParam,
+    SaleParamTez,
+    originateFixedPriceTezSale
+} from '../src/nft-contracts';
+import {
+    BalanceOfRequest,
+    transfer,
+    addOperator,
+    removeOperator
+} from '../src/fa2-interface';
+import { originateInspector, queryBalances } from './fa2-balance-inspector';
+
+jest.setTimeout(180000); // 3 minutes
+
+const nat1 = new BigNumber(1);
+
+describe('test market (test_case_1)', () => {
+    let tezos: TestTz;
+    let nft: Contract;
+    let inspector: Contract;
+    let marketplace: Contract;
+    let marketAddress: address;
+    let bobAddress: address;
+    let aliceAddress: address
+    let tokenId: BigNumber;
+    let tokenMetadata: MichelsonMap<string, bytes>;
+    let salePrice: BigNumber;
+
+    beforeAll(async () => {
+        tezos = await bootstrap();
+        inspector = await originateInspector(tezos.bob);
+        const admin = await tezos.bob.signer.publicKeyHash();
+        nft = await originateNftFaucet(tezos.bob, admin);
+        marketplace = await originateFixedPriceTezSale(tezos.bob);
```

Comment:
It looks like `nft_market_case_1.test.ts` and `nft_market_case_1.test.ts` are identical except the fact that they are creating different type of the market contract. Consider using the same code for both tests and factor out market contract creation function. See `nft-contracts.test.ts` for examples. 

```typescript
describe.each([originateFixedPriceTezSale, originateFixedPriceTezSale])(
    'test market`, (originateMarketplace) => {
        ......
        
        beforeEach(async () => {
              ...
              marketplace = await originateMarketplace(tezos.bob);
              ...
        })
        .....
    }
)